UIKitTemplate/UIKitTemplate/Model/User.swift

```diff
@@ -0,0 +1,24 @@
+// User.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
```

Comment:
```suggestion
import Foundation
```
здесь без него вроде можно, если никаких нету типов не простых